src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -146,21 +148,19 @@ class FileField extends React.Component {
 
   onAttachmentIdChange = (index, value) => {
     if (!value) {
-      this.props.onChange(
-        _.unset([index, 'attachmentId'], this.props.formData),
-      );
+      this.props.onChange(unset([index, 'attachmentId'], this.props.formData));
```

Comment:
I think `unset` is fine - the alternative would be a `delete`, and that would lead to more refactoring in this function which is out of scope.